elements/ul.html

```diff
@@ -0,0 +1,46 @@
+<article class="article">
+  <header class="article-header">
+    <h2>
+      <code>
+        &lt;ul&gt;
+      </code>
+    </h2>
+    <div class="base-words">
+      <div class="base-word-line">
+        English Base: <strong>Unordered List</strong>
+      </div>
+      <div class="base-word-line">
+        Spanish translation/Traducción en Español: <strong></strong>
+      </div>
+    </div>
+  </header>
+  <div class="definition">
+    <div class="definition-box definition-spanish">
+      <h4>Definici&oacute;n</h4>
+      <p></p>
+    </div>
+    <div class="definition-box definition-english">
+      <h4>Definition</h4>
+      <p>Used to group a list of items where the order that they are in is not important.  The bullet style can be changed to a wide variety of objects from dots to dashes to squares or even pictures.</p>
```

Comment:
The clause "that they are in" is redundant and kinda just noise. I would remove it. Maybe just "... a list of items whose order is not important."